dspot/src/main/java/eu/stamp_project/utils/DSpotCache.java

```diff
@@ -0,0 +1,35 @@
+package eu.stamp_project.utils;
+
+import org.ehcache.Cache;
+import org.ehcache.CacheManager;
+import org.ehcache.config.builders.CacheConfigurationBuilder;
+import org.ehcache.config.builders.CacheManagerBuilder;
+import org.ehcache.config.builders.ResourcePoolsBuilder;
+
+import eu.stamp_project.test_framework.TestFrameworkSupport;
+import eu.stamp_project.utils.program.ConstantsProperties;
+
+public class DSpotCache {
+	private static Cache<String, TestFrameworkSupport> frameworkCache;
+	private static CacheManager cacheManager;
+
+	static {
+		cacheManager = CacheManagerBuilder.newCacheManagerBuilder()
+				.withCache("frameworkCache", CacheConfigurationBuilder.newCacheConfigurationBuilder(String.class,
+						TestFrameworkSupport.class, ResourcePoolsBuilder.heap(
+								Long.parseLong(ConstantsProperties.CACHE_SIZE.getDefaultValue()))).build())
+				.build(true);
+
+		frameworkCache = cacheManager.getCache("frameworkCache", String.class, TestFrameworkSupport.class);
+
+		// frameworkCache = cacheManager.createCache("methodCache",
```

Comment:
Could remove these comments?

Reply:
Removed, I've kept them as a code snippet for creating additional caches, but as I guess you want to have a clean code (and I have this code reference in a bookmark) there is not problem to remove them.